liquibase-standard/src/main/java/liquibase/command/CommandFactory.java

```diff
@@ -186,19 +190,41 @@ public SortedSet<CommandDefinition> getCommands(boolean includeInternal) {
         return Collections.unmodifiableSortedSet(commands);
 
     }
-
+    private static String findKeyInMapIgnoreCase(String key, Map<String, String[]> map) {
+        for (Map.Entry<String, String[]> mapEntry : map.entrySet()) {
+            String actualKey = mapEntry.getKey();
+            if (actualKey.equalsIgnoreCase(key)) {
+                return actualKey;
+            }
+        }
+        return null;
+    }
```

Comment:
This method already exists in liquibase.util.CollectionUtil#findKeyInMapIgnoreCase and should not be duplicated here.

Reply:
I refactored this to just use a case-insensitive Set.